src/randoop/FailureSet.java

```diff
@@ -5,17 +5,27 @@
 import java.util.List;
 import java.util.Set;
 
+import randoop.operation.MethodCall;
+import randoop.sequence.ExecutableSequence;
+import randoop.sequence.MutableSequence;
+import randoop.sequence.MutableVariable;
+import randoop.sequence.Statement;
+import randoop.sequence.Variable;
+
 public class FailureSet {
   
   private Set<Failure> failures = new LinkedHashSet<Failure>();
   
   public static class Failure {
-    public final StatementKind st;
+    
+    public final Statement st;
```

Comment:
Sorry, didn't think about the confusion between renames and type substitutions.  This is the latter.